app/stockpile_api.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, services):
         await self.services.get('auth_svc').check_permissions(request)
         ability = dict(await request.json())
         ability_dirs = ['plugins/%s/data/abilities' % p.name.lower()
-                        for p in self.services.get('app_svc').get_plugins()
+                        for p in self.services.get('data_svc').locate('plugins', match=dict(enabled=True))
```

Comment:
shouldn't enabled get passed here from the hook.py setting (line 7)

Reply:
no, in this case we have 2 separate things..
1) the stockpile plugin is enabled/disabled (per the hook.py)
2) the stockpile plugin wants access to all enabled plugins (this line here)

Reply:
ah sweet thanks I see